### PR TITLE
Example notebook on how to check some conjectures

### LINKED ISSUE #141: Triangle (7, 8, 15)
The billiard in a rational polygon has the property that any trajectory that hits a side perpendicularly is either a saddle connection or periodic. In other words, this trajectory belongs to either a cylinder or the boundary of a cylinder in the translation surface unfolding.

The (7, 8, 15) triangle is one of the smallest triangles for which one of the directions orthogonal to the sides is not completely periodic. It would be nice to have this example in the documentation.

### PR DESCRIPTION
**Pull request by saraedum**
Fixes #141.

##### TODO

* [x] ~~Make the vue-flatsurf widget output work in our documentation. This is too hard to do at the moment.~~

### CODE REVIEW

File: doc/examples/boshernitzan_conjecture.md [hunk 1]
```diff
@@ -0,0 +1,354 @@
...
+
+We unfold this triangle and obtain a translation surface.
+
+```{code-cell} ipython3
+from flatsurf import similarity_surfaces
+S = similarity_surfaces.billiard(Δ).minimal_cover(cover_type="translation")
+S.plot()
+```
+
+Currently, the builtin plot is not terribly useful here. We can visualize this unfolding if [ipyvue-flatsurf](https://github.com/flatsurf/ipyvue-flatsurf) is available.
```
Comment (saraedum) [inline doc/examples/boshernitzan_conjecture.md:68]: Is there a better (easy) way in sage-flatsurf to plot this one?

File: doc/examples/boshernitzan_conjecture.md [hunk 2]
```diff
@@ -0,0 +1,354 @@
...
+
+```{code-cell} ipython3
+from flatsurf.geometry.pyflatsurf_conversion import to_pyflatsurf
+
+F = to_pyflatsurf(S)
+```
+
+##### A Unique Large Edge
+
+We start by retriangulating our surface. Namely, we want to obtain a single *large edge* for the flow direction, i.e., a unique edge that is wider (perpendicular to the flow direction) than all the other edges.
```
Comment (saraedum) [inline doc/examples/boshernitzan_conjecture.md:186]: **TODO** This is misleading.

File: doc/examples/boshernitzan_conjecture.md [hunk 3]
```diff
@@ -0,0 +1,354 @@
...
+Unfortunately, we cannot display a plot of such a surface since it is not a real translation surface anymore. Some of the edges (the ones is direction of the flow) have been collapsed, see [#62](https://github.com/flatsurf/vue-flatsurf/issues/62).
+
+```{code-cell} ipython3
+large = [e for e in F.edges() if F.vertical().large(e.positive())][0].negative()
+large
+```
+
+##### Flowing from the Large Edge to Construct an Interval Exchange Transformation
+
+We (unfortunately do not) see in the above plot how the half edges on the left get shuffled by to be become the half edges on the right if we follow the flow across the ``large`` edge.
```
Comment (saraedum) [inline doc/examples/boshernitzan_conjecture.md:207]:
```suggestion
We (unfortunately do not) see in the above plot how the half edges on the left get shuffled to be become the half edges on the right if we follow the flow across the ``large`` edge.
```

File: doc/examples/boshernitzan_conjecture.md [hunk 4]
```diff
@@ -0,0 +1,354 @@
...
+iet.swap()
+iet.zorichInduction()
+iet.swap()
+iet.zorichInduction()
+iet.swap()
+iet.zorichInduction()
+iet
+```
+
+Therefore, we can simplify the interval exchange transformation by dropping thop label `f`.
```
Comment (saraedum) [inline doc/examples/boshernitzan_conjecture.md:262]:
```suggestion
Therefore, we can simplify the interval exchange transformation by dropping the label `f`.
```

**Comment by saraedum** [comment]
@videlec it's been a while since we worked on this. Could you have a look and let me know what you think needs to be improved here?

**Comment by videlec** [comment]
The third cell containing
```
from ipyvue_flatsurf import Widget

cylinders = [c for c in D.components() if c.cylinder()]
Widget(cylinders)
```
does not appear as a cell but rather as part of a big text cell.

**Comment by videlec** [comment]
The number field used for the triangle has generator `c` which is confusing in this context as we have `a, b, c` for the angles...

**Pull request merged**